opencog/persist/CMakeLists.txt

```diff
@@ -7,4 +7,6 @@ IF (GUILE_FOUND)
 	ADD_SUBDIRECTORY (guile)
 ENDIF (GUILE_FOUND)
 
+ADD_SUBDIRECTORY (load_scm)
+
```

Comment:
It would be nice to have a `README` in the `load_scm` directory that explained what the code is for, and why it was created.